package.json

```
@@ -55,6 +55,9 @@
       "browser": "./bundles/web.bundle.min.js"
     }
   },
+  "bin": {
+    "ar-io": "./lib/esm/cli/cli.js"
```

```suggestion
    "ar.io": "./lib/esm/cli/cli.js"
```
(if possible, need to test)